client/js/kernel.js

```diff
@@ -43,17 +54,14 @@ var Services = require('jupyter-js-services');
                 kernel.statusChanged.connect(function(_kernel, status) {
                     clearTimeout(debounced);
                     debounced = setTimeout(function() {
-                        var isBusy = status === 'busy';
-                        $('.busy-indicator')
-                            .toggleClass('show', isBusy)
-                            // Prevent progress animation when hidden by removing 'active' class.
-                            .find('.progress-bar')
-                                .toggleClass('active', isBusy);
+                        var isBusy = status === Services.KernelStatus.Busy;
```

Comment:
I thought `Services.KernelStatus` doesn't exist anymore in latest **jupyter-js-services**. I had previously changed all references to use string values instead.